backend/main.go

```diff
@@ -408,13 +423,25 @@ func App() *cli.App {
 						SmtpPassword: config.Mailer.Password,
 					})
 
+					attachments := []Attachment{}
+					if (icsContent != nil){
```

Comment:
Please run `go fmt ./...` first, to make sure your code align with Go's formatting standards.